exercises/connect/src/main/java/Connect.java

```diff
@@ -0,0 +1,5 @@
+class Connect {
+    String findWinner(String[] board) {
+        throw new UnsupportedOperationException("Delete this statement and write your own implementation.");
+    }
+}
```

Comment:
Add new line and use the standard template for stub files for exercises with difficulty > 4.